Classes/Providers/OVP/OVPMediaProvider.swift

```diff
@@ -213,12 +221,23 @@ public class OVPMediaProvider: MediaEntryProvider {
                         mediaSources.append(mediaSource)
                     })
                     
+                    var metaDataItems: [(name: String, value: String)] = [(name: String, value: String)]()
+
+                    metadataList.forEach({ (meta:OVPMetadata) in
```

Comment:
I think this multi-level forEach is less readable than a plain old for-in loop.

```
for meta in metadataList {
...etc...
}
```

Also I'd move this loop to a method.